sagemaker-clarify/text_explainability/text_explainability.ipynb

```diff
@@ -7,20 +7,30 @@
     "# Explaining text sentiment analysis using SageMaker Clarify"
```

Comment:
<p>maybe you can annotate the types in the function and add a docstring showing what shap_explanations data look like? a "head" from them showing the data structure would help.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3793/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>